docs/csharp/tuples.md

```diff
@@ -85,6 +81,9 @@ unnamed tuple:
 
 [!code-csharp[UnnamedTuple](../../samples/snippets/csharp/tuples/tuples/program.cs#01_UnNamedTuple "Unnamed tuple")]
 
+The tuple in the previous example was initialized using literal constants and
+won't have field names created using *Tuple field name projections* in C# 7.1.
```

Comment:
>field [](start = 11, length = 5)

General terminology: we refer to "elements" of tuples, and the "cardinality" of tuples (for the number of elements).
ValueTuple on the other hand has fields and an arity (from 0 to 8).